controllers/login_controller.js

```diff
@@ -1,72 +1,29 @@
-const bcrypt = require('bcrypt');
-const dbHandler = require('../data/dbHandler');
-const {constructResponse} = require('../utils/utils.js');
+const loginService = require('../services/loginService');
```

Comment:
please keep the `camelCase` convention and change the file name from snake_case to camelCase